### PR TITLE
improve the `functions and expression` section

### PR DESCRIPTION
**Pull request by whitewum**
Ready to review

### CODE REVIEW

**Review by Amber1990Zhang — DISMISSED** [review]
pls add `;` to the end of each query

File: docs-2.0/3.ngql-guide/6.functions-and-expressions/7.count.md [hunk 1]
```diff
@@ -1,14 +1,31 @@
...
-You can use `count(*)` and `GROUP BY` together to group and count the number of specific values.
+## Syntax
 
-### Example 1
+```
+count({expr | *})
+```
+
+- count(*) returns the number of rows (including NULL),
+- count(expr) return non-NULL values return by an expression.
```
[the remark this answers is not in the document]
Comment (whitewum) [reply docs-2.0/3.ngql-guide/6.functions-and-expressions/7.count.md:15]: count(expr) will not sum NULL values, it only sum non-NULL values.

File: docs-2.0/3.ngql-guide/6.functions-and-expressions/10.collect.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# collect()
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/6.functions-and-expressions/10.collect.md:1]: Plz add semi-colons after each query.

File: docs-2.0/3.ngql-guide/6.functions-and-expressions/10.collect.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+```ngql
+nebula> UNWIND ["c", "b", "a", "d" ] AS p WITH p AS q ORDER BY q DESC LIMIT 3 RETURN collect(q)
++-----------------+
+| COLLECT(q)      |
++-----------------+
+| ["d", "c", "b"] |
++-----------------+
+```
+
+This example aggregates all players' name by their ages.
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/6.functions-and-expressions/10.collect.md:50]: names

**Review by randomJoe211 — APPROVED** [review]
Now we have collect 👍

**Review by Amber1990Zhang — APPROVED** [review]
pls add `;` to the end of each query

**Review by Amber1990Zhang — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**